dsl/src/main/java/tech/pegasys/peeps/node/Web3ProviderConfiguration.java

```diff
@@ -110,4 +114,8 @@ public boolean isPrivacyEnabled() {
   public Optional<String> getPrivacyMarkerSigningPrivateKeyFile() {
     return Optional.ofNullable(privacyMarkerSigningPrivateKeyFile);
   }
+
+  public Optional<SignerConfiguration> getWallet() {
```

Comment:
Does this mean for Besu? Does now require a EthSigner to provide a Wallet? Is this option just ignored for Besu?

Reply:
If you configure a Besu to do this, we we'll copy a wallet into besu ... so don't do it ;)